azext_iot/central/_help.py

```diff
@@ -253,6 +254,68 @@ def _load_central_users_help():
     """
 
 
+def _load_central_api_token_help():
+    helps[
+        "iot central app api-token"
+    ] = """
+        type: group
+        short-summary: Create and Manage API tokens .
+    """
+
+    helps[
+        "iot central app api-token create"
+    ] = """
+        type: command
+        short-summary: Create a new API token in the application
+        examples:
+        - name: Add new API token
+          text: >
+            az iot central app api-token create
+            --token-id {tokenId}
+            --app-id {appId}
+            --role admin
+
+    """
+    helps[
+        "iot central app api-token show"
+    ] = """
+    type: command
+    short-summary: Get an API token by ID
+    examples:
+      - name: Get API token
+        text: >
+          az iot central app api-token show
+          --app-id {appid}
+          --token-id {tokenId}
```

Comment:
Can we double check that this API actually returns a token to the developer? If not - lets remove it, I think it misleads our customers into thinking they can programmatically fetch a token. This call essentially just checks for existence of the token.

Let me know if PM has strong desire to keep this, we can work with them internally

Reply:
@prbans @valluriraj The use case for this is to list the tokens you've generated, and then be able to revoke them if needed. We can talk to Vivian if more info is needed. Let's keep parity with the API.